docs/components/compact.md

```diff
@@ -64,9 +64,18 @@ your Prometheus instances, so that the compactor will be able to group blocks by
 By _persistent_, we mean that one Prometheus instance must keep the same labels if it restarts, so that the compactor will keep
 compacting blocks from an instance even when a Prometheus instance goes down for some time.
 
+## Block Deletion
+
+Depending on the Object Storage provider like S3, GCS, cepth hdfs etc; the storage can be strongly consistent or eventually consistent. This poses a challenge to co-ordinate readers and writers of the block.
```

Comment:
What is `cepth`? (: 

```suggestion
Depending on the Object Storage provider like S3, GCS, Ceph etc; the storage can be strongly consistent or eventually consistent. This poses a challenge to co-ordinate readers and writers of the block.
```

HDFS is not an object storage.

Reply:
Sorry for being super picky here, but I hope you can learn from that (for example, to not be picky xD), but:

`the storage can be strongly consistent or eventually consistent.` is not technically true. It can be sometimes never consistent in failure scenarios ;p

Reply:
Maybe let's put it `we can divide those storages into strongly or eventually consistent.

Reply:
i have made this change now. thanks for the suggestion.